frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -314,6 +314,18 @@ export const Roles = defineMessages({
 export const getRole = (roleId: string | number): MessageDescriptor =>
   getOrThrowError(Roles, roleId, `Invalid role '${roleId}'`);
 
+export const ClassificationRoles = defineMessages({
+  [Role.Admin]: {
+    defaultMessage: "Administrator",
+    description: "The name of the Administrator user role.",
+  },
+});
+
+export const getClassificationRoles = (
+  roleId: string | number,
+): MessageDescriptor =>
+  getOrThrowError(ClassificationRoles, roleId, `Invalid role '${roleId}'`);
```

Comment:
ClassificationRoles is actually unrelated to user roles such as Admin. I think you can get away without any changes to localizedConstants in this PR.

Reply:
I am not pretty sure how I can bring classification roles to localizedConstnts without enum. Any help please?

Reply:
Good question. As an alternative to enums, you could reference the keystring. But again, I don't think you need to add anything to this file for this PR. The name field of ClassificationRole is already translated in the database, and the rest of the content is pretty specific to the RoleAndSalary form.